src/index.html

```diff
@@ -2,17 +2,99 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Product cards</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
```

Comment:
The `crossorigin` attribute should be part of the `<link>` tag on line 9. It should be written as `<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">`.